bciers/apps/administration/app/data/jsonSchema/contact.ts

```diff
@@ -129,6 +139,12 @@ export const contactsUiSchema = {
       "ui:widget": "ComboBox",
       "ui:placeholder": "Select the user",
     },
+    places_assigned: {
+      "ui:options": {
+        note: "You cannot delete this contact unless you replace them with other contact(s) in the place(s) above.",
+      },
+      "ui:ArrayFieldTemplate": InlineArrayFieldTemplate,
```

Comment:
nit: this: `"ui:classNames": "[&>div:last-child]:w-2/3",` will make your note component even more beautiful.